dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -34,10 +34,10 @@
     %p Edit Markdown:
 
     %textarea#level_markdown_textarea
-    #markdown-preview
+    #level_markdown_textarea_preview
 
   .markdown.teacher
     %p Edit Teacher Only Markdown:
 
     %textarea#level_teacher_markdown_textarea
-    #teacher-markdown-preview
+    #level_teacher_markdown_textarea_preview
```

Comment:
The issue here is that when I do
`= f.text_area 'teacher_markdown' ...`

I end up with an element id of `level_teacher_markdown` (so Ruby apparently already violates our style).
As a result I want to use `_preview` as my suffix rather than `-preview`. Though the element here is manually created instead of created via `f.text_area` like the others, I wanted to be consistent with the others.
You'll also notice that this file is already using snake_case on line 42.